lib/event_tracer/dynamo_db_log_worker.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+module EventTracer
+  class DynamoDBLogWorker
+    include ::Sidekiq::Worker
```

Comment:
is our impl here similar to TC & MC's?

i.e. is this worker useful for them? 

also will upgrading ET to this latest version cause issues due to clashing names?

_**technically** no since their version would override & "monkey-patch" this version of dynamo sidekiq logger_

Reply:
_😢  the sidekiq worker & logger are so tightly coupled tgt_

Reply:
Yep! They copied over. Yes.... very tightly coupled.

Reply:
Hm. Just one awkward thing. TC named the class `DynamoDbClient` while MC, GH named it `DynamoDBClient`. Alfred is okay because it is defined within the LogWorker. 

So the same pseudo-backwards-compatible situation here. TC just needs to be careful to rename the class if they remove their worker.

Reply:
ok cool; cc: @weikeat-wong for future reference